cmd/login/cmd.go

```diff
@@ -164,6 +169,21 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
+	// Load the configuration file:
+	cfg, err := config.Load()
+	if err != nil {
+		r.Reporter.Errorf("Failed to load config file: %v", err)
+		os.Exit(1)
+	}
+	if cfg == nil {
+		cfg = new(config.Config)
+	}
+
+	if (cfg.FedRAMP || fedramp.HasFlag(cmd) || fedramp.IsGovRegion(arguments.GetRegion())) && (args.useAuthCode || args.useDeviceCode) {
```

Comment:
We want to avoid using the stored configuration for this check. It will prevent someone from switching between govcloud and commercial environments if `"fedramp": "true"` is stored in their configuration.

Reply:
@tirthct @tylercreller Can we combine this statement into a function and write a unit test for it e.g something like `fedramp.IsLoginMethodSupported()`

cheers,

Rob

Reply:
@robpblake turns out we don't need the extra conditions except for fedramp.HasFlag() check. So just kept that instead of creating a whole new method.
Fixed the condition